app/models/partners/item_request.rb

```diff
@@ -37,11 +37,8 @@ def request_unit_is_supported
     end
 
     def name_with_unit(quantity_override = nil)
-      if Flipper.enabled?(:enable_packs) && request_unit.present?
-        "#{item&.name || name} - #{request_unit.pluralize(quantity_override || quantity.to_i)}"
-      else
-        item&.name || name
-      end
+      unit_text = request_unit.present? ? request_unit.pluralize(quantity_override || quantity.to_i) : "items"
```

Comment:
This changes the behavior in that it adds " - items" where it didn't do that before.

@cielf thoughts?

Reply:
I think you're right -- the "items" is unneeded.